code/src/reducers/game.js

```diff
@@ -0,0 +1,44 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+export const game = createSlice({
+    name: 'game',
+    initialState: {
+        username: '',
+        description: '',
+        direction: [],
+        coordinates: '',
+        history: [],
+        fetching: false,
+        timer: false
+    },
+    reducers: {
+        generateGame: (store, action) => {
+            //Update direction, description, coordinates after fetch.
+            store.direction = action.payload.actions;   
+			store.description = action.payload.description;
+            store.coordinates = action.payload.coordinates;
+            
+            //Saves direction, description, coordinates after fetch in history.
+            store.history = [...store.history, action.payload]; 
+        },
```

Comment:
Does this history-thingy do anything? We could not se a back-button or something that uses this?